sdk/storage/src/table/clients/table_client.rs

```diff
@@ -193,3 +206,253 @@ mod integration_tests {
         // TODO: Validate that the entity was inserted
     }
 }
+
+///////////////////////////////////////////////////////////////////////
```

Comment:
We don't use these style of comments so this should be removed.

Reply:
Done.